### ISSUE TITLE
Create a Google Slides template for TEC Comunity Call Updates

### ISSUE THREAD

**Issue by markoprljic** [CommonsBuild/content-distribution#22]
Take this doc and turn it into Google slides https://docs.google.com/document/d/1hdGDugNqB33lsWn7eTWIeIt2CQ5HHTqvdYKru5wquG4/edit#heading=h.j2t16l2tpic9

**Comment by markoprljic** [comment]
@CommonsBuild/community-stewards Here's the Community Call Slides template, please review https://docs.google.com/presentation/d/1msWeZyBOURMEMqF_WtsmYqaBnBCQz7oKkKLp2xxpwIE/edit?usp=sharing

**Comment by chuygarcia92** [comment]
This is great!!!!! @markoprljic I would add two separate text boxes in the updates slides. One of these would be for topics in the agenda and the other one for annotations that may be brought up during the call.

**Comment by markoprljic** [comment]
It's up to you guys to edit your slides as you see fit. This is the framework and the template, you take it from here @CommonsBuild/community-stewards

**Comment by tamarandom** [comment]
Updated the Agenda in the Community Sync cal invite! <-- @iviangita this means the URL should stay the same each week. If we are planning to archive can we copy the archive to another deck and update this same one each week. Let me know if that's not clear enough. ;)

Added to Stewards' agenda this week as a last reminder to update it before Thursday and take into consideration @chuygarcia92 and others' feedback.

@markoprljic no reason to keep this open anymore, right?

**Comment by markoprljic** [comment]
@tamarandom No reason to keep it open, I moved it to Published.

Suggestion to use the deck
![Image 2021-04-26 at 1 37 33 pm](https://user-images.githubusercontent.com/5704285/116076624-c0814500-a694-11eb-8a6c-c8f0000a4250.jpg)

**Comment by iviangita** [comment]
Hi @tamarandom @markoprljic I created a copy of the deck of last Thursday call and archived it so we will only use the same URL every week. Then I created a tracker of the archived decks for future reference. Hope this is okay? Let me know if you have other suggestions. Thanks! 

Archived agenda tracker: https://docs.google.com/spreadsheets/d/1UYCLstL4i-kzYPwenEP0UWGhW7_bQgE4THQqeL7yQqM/edit?usp=sharing
